scripts/installscripts/buster-install-default.sh

```diff
@@ -543,19 +590,20 @@ main_install() {
     sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
 
     # Get github code
-    cd /home/pi/ || exit
+    cd "${HOME_DIR}" || exit
     git clone https://github.com/MiczFlor/RPi-Jukebox-RFID.git --branch "${GIT_BRANCH}"
 
     # add used git branch and commit hash to version file
     USED_BRANCH="$(git --git-dir=${jukebox_dir}/.git rev-parse --abbrev-ref HEAD)"
-    sudo sed -i 's/%GIT_BRANCH%/'"$USED_BRANCH"'/' ${jukebox_dir}/settings/version
+    sudo sed -i 's/%GIT_BRANCH%/'"$USED_BRANCH"'/' "${jukebox_dir}"/settings/version
 
     # add git commit hash to version file
```

Comment:
The comment is redundant to line 596

Reply:
Yes, I agree. Checking for redundant comments was outside of the scope of this PR though. ;)